content/documentation/install-uninstall.md

```diff
@@ -6,61 +6,57 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
 ---
 
 {{< q-class "box warning" >}}
-- Quire is in a closed beta. Installation will not work unless you have [requested and been given access](https://goo.gl/forms/Ih4SOHje0Z9dm3C73).
+- Quire is in a limited beta, © J. Paul Getty Trust, and not yet released as open source software. For a free license to use Quire for your publication projects (and for additional access to support from Getty staff) [please sign up](https://goo.gl/forms/Ih4SOHje0Z9dm3C73).
 {{< /q-class >}}
 
 ## macOS Installation
 
-Open your Terminal {{< q-def "command-line shell" >}} (found in your Applications/Utilities folder or by pressing Command–Space Bar and type “Terminal” ) and follow the steps below to first install the support software for Quire, and then Quire itself. If you are new to the command-line, read our tutorial [*Working in a Command-line Shell*](/learn/tutorial/#1-working-in-a-command-line-shell).
+Installing and running Quire requires using the Terminal {{< q-def "command-line shell" >}}. Open it from your Applications/Utilities folder or by pressing Command–Space Bar and typing “Terminal”. If you are new to the command-line, read our tutorial [*Working in a Command-line Shell*](/learn/tutorial/#1-working-in-a-command-line-shell).
 
-1. **Apple’s Xcode** is a set of developer tools for your mac. More information can be found [here](https://en.wikipedia.org/wiki/Xcode). Install Apple's Xcode by copying and pasting the following command and pressing enter. If Xcode is not already installed, an additional alert notification will pop up. Click “Install” and follow the prompts.
+### Quick Install
 
-    ```text
-    xcode-select --install
-    ```
+If you’re eager to get started, this will install the complete Quire package but without e-book or PDF output capability. These may be added later, by following steps 2–4 in the Full Install guidelines below.
 
-2. **Node.js** enables you to run javascript on your computer. Visit the Node.js site, and download and install the current LTS (long-term support) version: [https://nodejs.org](https://nodejs.org).
+1. Install the **LTS** version of Node.js: [https://nodejs.org](https://nodejs.org)
+2. In Terminal, install the Quire CLI with: `npm install --global @thegetty/quire-cli`
+3. Confirm installation: `quire --help`
```

Comment:
I changed this to `--help` because I actually thought that would be a good way to teach people that this command brings up a list of all the quire commands. But what do you think? Here's what it looks like now, and we could definitely work with Hannah and Matthew to add to or expand on this:

```
galbers@GT29043 quire-docs % quire --help
Usage: quire [options] [command]

Options:
  -V, --version                output the version number
  -v, --verbose                log verbose output
  -f, --file                   Add Filename and optional new filepath
  -e, --env                    Add environment variable
  -h, --help                   output usage information

Commands:
  new <projectName>            Create a new Quire project in the current directory.
  preview [options]            Run the preview server in the current directory
  install                      Install this project's theme dependencies
  site|build [options] [env]   Run the build command in the current directory
  pdf [options] [file] [env]   Generate a PDF version of the current project
  epub [options] [file] [env]  Generate an EPUB version of the current project
  mobi [options] [file] [env]  Generate an MOBI version of the current project
  template [options] [type]    Download templates to customize your file output (Currently only EPUB supported)
  debug                        Development use only - log info about current project
```

Reply:
I agree! I would suggest editing to say something along the lines of:

Step. 3 - Run the command `quire --help.` This will give you a list of commands that will help you get started using and navigating Quire. More can be found in the Quire Commands section of the docs. And if you get "command not found" it means it was not installed correctly. See Troubleshooting section, etc. 

When I used Quire for the first time and ran the version command as the final install step, I was so thrilled to know I did it correctly and didn't get the error message. It gave me a sense of accomplishment and good momentum moving forward. Then they'd also know that, oops, if they did get an error message they can check Troubleshooting.

Maybe we include a note in Troubleshooting section directing people to the forum.